content/applications/hr/referrals.rst

```diff
@@ -0,0 +1,703 @@
+are available to spend.
+
+To see the various referrals and the points earned for each, click on one of the buttons directly
+beneath the avatars. The options are :guilabel:`Referrals`, :guilabel:`Ongoing`, and
+:guilabel:`Successful`.
+
+The number of referrals for each category is displayed above the word. The total number of referrals
+(the ongoing and successful referrals combined) appear above the `Referrals` button. The current
+number of referrals who have not yet been hired or refused, appear above the :guilabel:`Ongoing`
+button. The number of referrals who were successfully hire appear above the :guilabel:`Successful`
+button.
+
+.. note::
+   To view the referrals associated with each of the categories, :guilabel:`Referrals`,
+   :guilabel:`Ongoing`, and :guilabel:`Successful`, click on the corresponding button. All the
+   referrals for that specific category are displayed.
+
+Referrals
+---------
+
+To see all the referrals (both ongoing and successful) click the :guilabel:`Referrals` button. A
+:guilabel:`My Referral` window loads. This screen displays all the referrals, with each referral in
+its own referral card.
+
+A successful referral has a green line down the left side of the card, and displays a `✓ Hired`
+image in the top right corner of the card. Referrals that are in process display an `In Progress`
+image in the top right corner.
+
+Each referral card lists the name of the applicant, the department the job position is in, the name
+of the job position, the person responsible for filling the role (recruiter), and the points earned.
+
+Points
+------
+
+The points that can be earned for a referral are the same across all job positions. Each stage of
+the recruitment process has corresponding points assigned to it. The stages listed correspond to the
+stages configured in the *Recruitment* application.
+
+On each individual referral card, beneath the points earned is a bar chart that displays how many
+points have been earned out of the total possible points that can be earned if the applicant is
+hired.
+
+Beneath the bar chart is a list of the various recruitment stages and the points earned when the
+referral moves to that stage. If a stage has been achieved and the points have been earned, a green
+check mark appears next to the stage.
+
+The pre-configured point structure for referrals is as follows:
+
+- :guilabel:`Initial Qualification`: 1 point
+- :guilabel:`First Interview`: 20 pointss
+- :guilabel:`Second Interview`: 9 points
+- :guilabel:`Contract Proposal`: 5 points
+- :guilabel:`Contract Signed`: 50 points
+
+The total points earned for a referral that is hired is 85 points. Modifications of the points
+earned for each stage is done in the Recruitment application. Refer to this documentation to modify
+the points for each stage.
+
+.. _referrals/rewards:
+
+Rewards
+=======
+
+Create rewards
+--------------
+
+The rewards are the only configurations needed when setting up the Referrals application.
```

Comment:
I understand that this is the only *required* setup, but since there are other optional configurations, I don't know that you want to include this line.

Reply:
The reason I put this in, is that I thought it was important to explain that everything is ready to go out of the box, except this. That there are NO rewards you can give unless they are configured, but the entire app is ready to go from the start. Unlike other apps where you really should configure some optional stuff, this one really doesn't need it.